src/community/label_propagation.jl

```diff
@@ -0,0 +1,81 @@
+"""
+Community detection using the label propagation algorithm (see [Raghavan et al.](http://arxiv.org/abs/0709.2938)).
+`g`: imput Graph
+`ϵ`: proportion of unsatisfied nodes
+return : array containing vertex assignments
+"""
+function label_propagation(g::SimpleGraph, ϵ=1.0e-8)
+    n = nv(g)
+    label = collect(1:n)
+    runing_nodes = Set(vertices(g))
+
+    while length(runing_nodes) > n*ϵ
```

Comment:
Should we also have a maxiter parameter so that the loop doesn't run forever if it is impossible to satisfy all the vertices? does the process guarantee that the number of iterations is small for all graphs? Documenting that bound is important if we have it.